### ISSUE TITLE
0.1.10 - No source map is available warning

### ISSUE THREAD

**Issue by et** [tildeio/route-recognizer#88]
```
Warning: ignoring input sourcemap for bower_components/route-recognizer/dist/route-recognizer.js because ENOENT: no such file or directory, open '/redacted/tmp/concat_with_maps-input_base_path-QMOFZnGZ.tmp/0/bower_components/route-recognizer/dist/route-recognizer.js.map'
```
This is exactly like #44 (fixed in 0.1.9) but it is happening again in the latest release 0.1.10.
We started noticing this occurring exactly today. Coincidentally 0.1.10 was published a couple a hours ago that's why we suspect there was a regression.

**Comment by rwjblue** [comment]
Sorry about that! Looks like the sourcemap file was accidentally deleted in https://github.com/tildeio/route-recognizer/pull/75.

**Comment by rwjblue** [comment]
Should be fixed now with v0.1.11, please confirm and close...

**Comment by et** [comment]
Praise @rwjblue!!
Sending a virtual high five your way, this issue has been resolved. Thanks!
